ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -40,11 +40,11 @@ private[platform] object CompletionFromTransaction {
           commandId = commandId,
           transactionId = transactionId,
           applicationId = applicationId,
-          maybeStatus = Some(OkStatus),
-          maybeSubmissionId = maybeSubmissionId,
-          maybeDeduplicationOffset = maybeDeduplicationOffset,
-          maybeDeduplicationDurationSeconds = maybeDeduplicationDurationSeconds,
-          maybeDeduplicationDurationNanos = maybeDeduplicationDurationNanos,
+          optStatus = Some(OkStatus),
+          optSubmissionId = optSubmissionId,
+          optDeduplicationOffset = optDeduplicationOffset,
+          optDeduplicationDurationSeconds = optDeduplicationDurationSeconds,
+          optDeduplicationDurationNanos = optDeduplicationDurationNanos,
```

Comment:
(This is just to spare a few chars on some long identifiers without making them less clear)